rust/azure_iot_operations_connector/Cargo.toml

```diff
@@ -15,6 +15,7 @@ publish = true
 azure_iot_operations_protocol = { version = "0.9", path = "../azure_iot_operations_protocol", registry = "aio-sdks"  }
 azure_iot_operations_services = { version = "0.8", path = "../azure_iot_operations_services", registry = "aio-sdks", features = ["state_store", "schema_registry", "azure_device_registry"]  }
 azure_iot_operations_mqtt = { version = "0.9", path = "../azure_iot_operations_mqtt", registry = "aio-sdks"  }
+derive-getters = { version = "0.5.0", features = ["auto_copy_getters"] }
```

Comment:
If we're starting to use this in a few places, should we move it to the workspace?

Reply:
moved!